autonomy/cli/helpers/registry.py

```diff
@@ -168,12 +168,27 @@ def publish_service_ipfs(public_id: PublicId, package_path: Path) -> None:
     if not IS_IPFS_PLUGIN_INSTALLED:  # pragma: nocover
         raise RuntimeError("IPFS plugin not installed.")
 
-    ipfs_tool = IPFSTool(get_ipfs_node_multiaddr())
-    _, package_hash, _ = ipfs_tool.add(str(package_path.resolve()))
-    package_hash = to_v1(package_hash)
-    click.echo(
-        f'Service "{public_id.name}" successfully published on the IPFS registry.\n\tPublicId: {public_id}\n\tPackage hash: {package_hash}'
-    )
+    package_path = package_path.resolve()
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        temp_service_dir = Path(temp_dir, public_id.name)
+        temp_service_dir.mkdir()
+        shutil.copyfile(
```

Comment:
Is it mandatory to have a README.md file? will this fail otherwise?

Reply:
Yes, every package needs to have a readme file